package.json

```
@@ -0,0 +1,44 @@
+{
+    "author": "Chiitoi",
+    "dependencies": {
+        "@discordjs/core": "^0.5.2",
+        "@discordjs/rest": "^1.7.0",
+        "@discordjs/ws": "^0.8.1",
+        "@prisma/client": "^4.13.0",
+        "dayjs": "^1.11.7",
+        "discord-api-types": "^0.37.39",
+        "dotenv": "^16.0.3"
+    },
+    "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^5.59.0",
+        "@typescript-eslint/parser": "^5.59.0",
+        "eslint": "^8.38.0",
+        "prisma": "^4.13.0",
+        "rimraf": "^5.0.0",
+        "typescript": "^5.0.4"
+    },
+    "engines": {
+        "node": ">=18"
+    },
+    "imports": {
+        "#interactions": "./dist/interactions/index.js",
+        "#structs": "./dist/structs/index.js",
+        "#types/*": "./dist/types/*.d.ts",
+        "#utility": "./dist/utility/index.js"
+    },
+    "license": "MIT",
+    "main": "dist/index.js",
+    "name": "tori",
+    "scripts": {
+        "build": "npx tsc -b",
```

`tsc` should be installed by the `typescript` package. The use of `npx` is redundant and can be removed.
```suggestion
        "build": "tsc -b",
```